crates/object_detection/src/pose_detection.rs

```diff
@@ -222,6 +222,36 @@ impl PoseDetection {
             human_poses: poses.into(),
         })
     }
+
+    fn should_look_for_referee<T: TimeInterface>(
+        &mut self,
+        hardware_interface: &Arc<T>,
+        motion_command: &MotionCommand,
+        pose_detection_inactive_duration: &Duration,
+    ) -> bool {
+        let is_in_initial_with_pose_detection = matches!(
```

Comment:
```suggestion
        let should_look_for_referee = matches!(
```
or
```suggestion
        let behavior_requests_pose_detection = matches!(
```

Reply:
Done. But is again now tracked in #1093.